docs/developer-center/new-guides/quickstart.md

```diff
@@ -1,185 +1,966 @@
+      <td>MULTIPOLYGON (((-73.96187 40.58632, -73.96289 ...</td>
+      <td>1534.248133</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-73.99082 40.62694, -73.99170 ...</td>
+      <td>1311.667005</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-74.02851 40.64062, -74.02882 ...</td>
+      <td>1179.999175</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>900</td>
+      <td>MULTIPOLYGON (((-74.00110 40.60186, -74.00249 ...</td>
+      <td>1334.454790</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Great! Let's see the result in a map:
+
+
+```python
+from cartoframes.viz.helpers import color_continuous_layer
+
+Map(color_continuous_layer(isochrones_df, 'POPCY', 'Population'))
+```
+
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_pop"
+        src="https://cartoframes.carto.com/kuviz/2f8f2c92-f8ed-4d1d-8e09-04ef85ad80dd"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
 
-First, you have to upload the data used by your maps to CARTO using the [Dataset]() class:
 
-```py
-from cartoframes.data import Dataset
+We can see that the area of influence of the store on the right, it's the one with the largest population. Let's go a bit further and calculate and visualize the average revenue per person.
 
-Dataset(arlington_no_car_df).upload(table_name='arlington_ct_no_cars', credentials=creds, if_exists='replace')
-Dataset(bikeshare_df).upload(table_name='bikeshare_july_agg', credentials=creds, if_exists='replace')
+
+```python
+starbucks_df = starbucks_df.reset_index(drop=True)
+starbucks_df['rev_pop'] = starbucks_df['revenue']/isochrones_df['POPCY']
+Map(size_continuous_layer(starbucks_df, 'rev_pop', 'Revenue per person'))
 ```
 
-Now, let's add widgets so people are able to see some graphs about the information displayed and allow them to filter it. To do this, we only have to add `widget=True` to the helpers. Remember to check the [visualization guide]() to learn more about it.
+<div class="example-map">
+    <iframe
+        id="quickstart_guide_starbucks_brooklyn_rev_pop"
+        src="https://cartoframes.carto.com/kuviz/6b3e7bc2-cca2-4172-9f6e-73deb71cde03"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+
+As we can see, there are clearly 3 stores that have lower revenue per person. This insight will help us to focus on them in further analyses.
+
+To know more about discovering the data you want, check the [data discovery guide](). Check the [data enrichment guide]() to know more possibilities about enriching your data.
+
+### Publish and share your results
+
+To finish your work, you want to share the results with your teammates. Also, it would be great if you could allow them to play with the information. Let's do it!
```

Comment:
replace: "To finish your work, you want to share the results with your teammates. Also, it would be great if you could allow them to play with the information. Let's do it!"

with: "The final step in the workflow is to share this interactive map with your colleagues so they can explore the information on their own. Let's do it!"